src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -159,12 +161,27 @@ public String getClusterName() {
 
   @Override
   public List<String> getKeyspaces() {
-    return null; // TODO: implement me.
+    try {
+      return apiClient.listKeyspaces("");
+    } catch (ApiException ae) {
+      LOG.error("Failed to list keyspaces", ae);
+      return Collections.emptyList();
+    }
   }
 
   @Override
   public Set<Table> getTablesForKeyspace(String keyspace) throws ReaperException {
```

Comment:
Issue: Would be good to see a unit test for this too please :) Similar procedure to the other one, mock the call to the apiClient method and confirm that the processing in the rest of the method does what it should.

Reply:
I added both unit tests as requested.